docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -68,99 +73,111 @@ To develop and test the app, you'll install it on your [sandbox store](#configur
 | Sites & Routes | read-only | `store_sites_read_only` |
 | Storefront API Tokens | generate tokens | `store_storefront_api` |
 
-4. On the **App Supported Features** tab, indicate that this is a multi-storefront enabled app.
+4. On the **App Supported Features** tab, select the option that indicates this is a multi-storefront enabled app.
 
 5. Click **Update & Close**, then click **Confirm Update** to acknowledge the OAuth scopes you configured.
 
-Further steps in this guide require access to the Dev Portal, so keep it handy.
+Make sure to remain signed into the Dev Portal
 
 ## Configure Stripe
 
-We've designed this integration to use Stripe Connect so that your app can use one connection to submit payments to multiple merchants' previously configured Stripe accounts. See [Stripe Billing](https://support.bigcommerce.com/s/article/Connecting-Stripe-Payment-Gateway?language=en_US#billing) for merchant configuration instructions.
+Subscription Foundation uses Stripe Connect so that your app can use one signle connection to submit payments to multiple previously configured Stripe merchant accounts. See [Stripe Billing](https://support.bigcommerce.com/s/article/Connecting-Stripe-Payment-Gateway?language=en_US#billing) for merchant configuration instructions.
 
-The respective architectures of the Stripe and BigCommerce APIs on which this app relies require the app to make charges using the **merchant-specific Stripe account's public key** and the **app-specific Stripe account's secret key**. 
+The integration between Stripe and BigCommerce requires that the app uses two keys to issue charges: the **merchant-specific Stripe account's public key** and the **app-specific Stripe account's secret key**.
 
-This arrangement provides the following benefits:
-* The merchant can change the Stripe account they use without breaking the integration.
-* The merchant's own Stripe account stores all shopper payment history, subscription and one-time purchase alike. 
-* It limits the app's PCI compliance burden. 
+Using two separate keys provides the following benefits:
+* The merchant can change between Stripe accounts they use without breaking the integration.
+* The merchant's own Stripe account stores the entire shopper payment history, including subscription and one-time purchases. 
+* The usage of public and secret keys ensures that your configuration is PCI compliant.
+
+For development you will use two Stripe accounts:
```

Comment:
simplify verb tense

Reply:
also I don't love numbering these accounts unless we're going to say that the numbers are based on the order the developer creates the account.